src/privatesend-client.cpp

```diff
@@ -1188,28 +1177,36 @@ bool CPrivateSendClientSession::SubmitDenominate(CConnman& connman)
         return false;
     }
 
-    // lean towards "highest" branch but still mix via "lowest" one someties
-    bool fMixLowest = privateSendClient.nLiquidityProvider || (GetRandInt(4) == 0);
-    // Try to use only inputs with the same number of rounds, from low to high, or vice versa
-    int nLoopStep = fMixLowest ? 1 : -1;
-    // lean towards edges but still mix starting from the middle someties
-    // Note: liqudity providers always start from 0
-    bool fScanFromTheMiddle = (privateSendClient.nLiquidityProvider == 0) && (GetRandInt(4) == 0);
-
-    int nRoundStart = GetStartRound(fMixLowest, fScanFromTheMiddle);
-    int nRoundEdge = GetStartRound(fMixLowest, false);
+    std::vector< std::pair<int, size_t> > vecInputsByRounds;
+    // Note: liqudity providers are fine whith whatever numner of inputs they've got
```

Comment:
nit: s/whith/with/

Reply:
s/numner/number/

Reply:
omg, 2 typos in one sentence 🙈 fixed :)

Reply:
Actually 3 typos  :laughing: "liquidity"

Reply:
This is the old one actually https://github.com/dashpay/dash/pull/2278/files#diff-8bdec64a6a2adb83b9696e5ca6d5522bL1196 but yeah..